x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/all/exceptions/translations.ts

```diff
@@ -96,3 +96,38 @@ export const EXCEPTION_DELETE_ERROR = i18n.translate(
     defaultMessage: 'Error occurred deleting exception list',
   }
 );
+
+export const EXCEPTION_DELETE_SUCCESS = i18n.translate(
+  'xpack.securitySolution.detectionEngine.rules.all.exceptions.deleteSuccess',
+  {
+    defaultMessage: 'Exception list deleted successfully',
+  }
+);
+
+export const REFERENCE_MODAL_TITLE = i18n.translate(
+  'xpack.securitySolution.exceptions.referenceModalTitle',
+  {
+    defaultMessage: 'Remove exception list',
+  }
+);
+
+export const REFERENCE_MODAL_CANCEL_BUTTON = i18n.translate(
+  'xpack.securitySolution.exceptions.referenceModalCancelButton',
+  {
+    defaultMessage: 'Cancel',
+  }
+);
+
+export const REFERENCE_MODAL_CONFIRM_BUTTON = i18n.translate(
+  'xpack.securitySolution.exceptions.referenceModalDeleteButton',
+  {
+    defaultMessage: 'Remove exceptions list',
```

Comment:
Do we want the singular form here, or plural so we're matching the `exceptions_list` field name? I think we can go with the singular, but whatever is best for consistency. [Doesn't look like the docs](https://www.elastic.co/guide/en/security/current/exceptions-api-overview.html) reference them as either, but rather an `Exception Container`, but we're not using that terminology in the UI at the moment.

Reply:
I think I said it in my head so many times that I'm not sure which sounds right :) I switched them to the singular for consistency. I think that does actually make most sense. If I recall we had wanted to change `exceptions_list` field to `exception_lists` at one point.